README.md

```diff
@@ -5,41 +5,103 @@
   <br />
 </p>
 
-# Sentry Rust
+# Sentry SDK for Rust
 
 [![Build Status](https://travis-ci.com/getsentry/sentry-rust.svg?branch=master)](https://travis-ci.com/getsentry/sentry-rust)
-[![Crates.io](https://img.shields.io/crates/v/sentry.svg?style=flat)](https://crates.io/crates/sentry)
-
-This workspace contains various crates that provide support for logging events
-and errors / panics to the [Sentry](https://sentry.io/) error logging service.
-
-- [sentry](./sentry) The main `sentry` crate aimed at application users that
-  want to log events to sentry.
-- [sentry-actix](./sentry-actix) An integration for the `actix-web (0.7)`
-  framework.
-- [sentry-core](./sentry-core) The core of `sentry`, which can be used to
-  instrument code, and to write integrations that generate events or hook into
-  event processing.
-- [sentry-types](./sentry-types) Contains types for the Sentry v7 protocol as
-  well as other common types.
-
-**Note**: Until the _1.0_ release, the crates in this repository are considered work in
-progress and do not follow semver semantics. Between minor releases, we might
-occasionally introduce breaking changes while we are exploring the best API and
-adding new features.
+
+This workspace contains various crates that provide support for logging events and errors / panics to the
+[Sentry](https://sentry.io/) error logging service.
+
+- [sentry](./sentry) [![crates.io](https://img.shields.io/crates/v/sentry.svg)](https://crates.io/crates/sentry)
+  [![docs.rs](https://docs.rs/sentry/badge.svg)](https://docs.rs/sentry)
+
+  The main `sentry` crate aimed at application users that want to log events to sentry.
+
+- [sentry-actix](./sentry-actix)
+  [![crates.io](https://img.shields.io/crates/v/sentry-actix.svg)](https://crates.io/crates/sentry-actix)
+  [![docs.rs](https://docs.rs/sentry-actix/badge.svg)](https://docs.rs/sentry-actix)
+
+  An integration for the `actix-web (0.7)` framework.
```

Comment:
I recently added the list of defaults here: https://github.com/getsentry/sentry-rust/blob/0a361de079ec667026da34aec92f2fb33a17c0d6/sentry/src/defaults.rs#L15-L25

And yes, I’m afraid all of this will get out of sync at some point. It is surprisingly tedious to keep all of this up to date.